tests/spec_tests/query_tests/select.yml

```diff
@@ -109,3 +109,9 @@ tests:
     result:
       - {'a': {'_id': 0, 'a': 1, 'b': 1}, '': {'a': 1}}
       - {'a': {'_id': 1, 'a': 2, 'b': 2}, '': {'a': 2}}
+
+  - description: SELECT many fields with some aliases containing dots and starting with dollars
```

Comment:
This should not be a spec test, it should be an e2e test. It is not demonstrating anything relevant to the specification, rather it is demonstrating an implementation-specific issue based on the mongodb back-end. Can you move this to the `e2e` test suite?

Reply:
moved